metpy/calc/thermo.py

```diff
@@ -2155,3 +2155,63 @@ def dewpoint_from_specific_humidity(specific_humidity, temperature, pressure):
     return dewpoint_rh(temperature, relative_humidity_from_specific_humidity(specific_humidity,
                                                                              temperature,
                                                                              pressure))
+
+
+@exporter.export
+@preprocess_xarray
+@check_units('[length]/[time]', '[mass]/[volume]')
+def vertical_velocity_height_to_pressure_hydrostatic(w, density):
+    r"""Calculate omega from w assuming hydrostatic conditions.
```

Comment:
I'd probably add some more description of this converts vertical velocity (w) to a velocity in terms of pressure rate of change (omega) for the uninitiated.

Reply:
Will do!